tests/foreman/ui/test_settings.py

```diff
@@ -175,6 +175,60 @@ def test_negative_validate_error_message(session, set_original_property_value):
                 assert is_valid_error_message(str(context.value))
 
 
+@tier2
+def test_positive_selectors(session):
+    """"Testing input for selectors: dropdown, text area, input box
+
+    :id: 529ddd3a-1271-4043-9006-eac436b08b11
+
+    :expectedresults: Successfully add value to different selectors
+
+    :CaseImportance: High
+    """
+    uuid_input = f'[ {gen_string("alpha")} ]'
+    random_input = gen_string('alpha')
+    uuid_duplicate_prop = 'host_dmi_uuid_duplicates'
+    reg_host_prop = 'register_hostname_fact'
+    dep_solve_prop = 'content_view_solve_dependencies'
+    with session:
+        # Drop down selector
+        session.settings.update(
+            'name = {}'.format(dep_solve_prop),
+            'Yes'
+        )
+        result = session.settings.read('name = {}'.format(dep_solve_prop))
+        assert result['table'][0]['Value'] == 'Yes'
+        # reset input value
+        session.settings.update(
```

Comment:
updating 'content_view_solve_dependencies' settings can lead to adverse effect on errata count in another test. ( there are many of them ) Here I would recommend preserving the earlier value and update that with API.   
e.g. If this UI test fails after updating solve_depedancy to 'Yes', It does not update to 'No' in that case it will create problems. You can solve this making yield test at module level and preserve the earlier values of add try-catch statement